metis-indexing/metis-indexing-client/src/main/java/eu/europeana/indexing/client/IndexingException.java

```diff
@@ -0,0 +1,9 @@
+package eu.europeana.indexing.client;
```

Comment:
This Exception class can be moved to the metis-common module under the exception package